packages/e2e-tests/src/specs/dashboard/adminMenu.js

```diff
@@ -27,9 +27,9 @@ describe('Admin Menu', () => {
 
     // Initial visit to `/` makes `Dashboard` link current in WP
     await page.hover('#menu-posts-web-story');
-    await await expect(
-      page.$('#menu-posts-web-story .current a')
-    ).resolves.toMatch('Dashboard');
+    await expect(page.$('#menu-posts-web-story .current a')).resolves.toMatch(
```

Comment:
😂 this test file is getting flagged by codecov. I can write a ticket to look into this (unless it's a known problem already)
![image](https://user-images.githubusercontent.com/22185279/149589455-6b4b08b0-a6cc-45c1-9a6d-3596e3d621d7.png)

Reply:
Updated the codecov config,  that should fix this. Thanks for flagging!